bgpd/bgp_vty.c

```diff
@@ -14199,7 +14199,7 @@ static int lcommunity_list_unset_vty(struct vty *vty, int argc,
 		vty_out(vty, "This config option is deprecated, and is scheduled for removal.\n");
 		vty_out(vty, "if you are using this please migrate to the below command.\n");
 		vty_out(vty, "'no bgp large-community-list <(1-99)|(100-500)|standard|expanded> <deny|permit> <LINE|AA:BB:CC>'\n");
-		zlog_warn("Deprecated option: 'no ip large-community-list <(1-99)|(100-500)|standard|expanded> <deny|permit> <LINE|AA:BB:CC>' being used");	
+		zlog_warn("Deprecated option: 'no ip large-community-list <(1-99)|(100-500)|standard|expanded> <deny|permit> <LINE|AA:BB:CC>' being used");
 	}
```

Comment:
no spaces extra,pls

Reply:
vscode just removed them automatically. or should I leave them as they were?

Reply:
Don't mix whitespace changes into commits that have "real" changes (except on the actual code you're touching, there you can fix small whitespace problems.)

https://github.com/FRRouting/frr/blob/master/doc/developer/workflow.rst#L549

In this case, this line should not be changed in this commit. If you want you can make the whitespace change a separate commit, cleanups are always welcome :)